duckduckgo/Android
# Feature/david/remove bottom bar

Task/Issue URL: https://app.asana.com/0/72649045549333/1177859689086266
Tech Design URL: 
CC: 

**Description**:
The Bottom Navigation experiment is now concluded and we've agreed to remove it for now. This PR removes all the code from the experiment.

**Steps to test this PR**:
1. Open the app
2. Ensure ""mb" variant is no longer appearing
3. There should be no traces of bottom navigation
4. Open Tab Switcher screen
5. There should be no traces of bottom navigation

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
